react/features/virtual-background/components/VirtualBackgroundPreview.js

```diff
@@ -78,6 +78,20 @@ class VirtualBackgroundPreview extends PureComponent<Props, State> {
         };
     }
 
+    /**
+     * Destroys the jitsiTrack object.
+     *
+     * @param {Object} jitsiTrack - The track that needs to be disposed.
+     * After disposing it stop stream is needed.
+     * @returns {Promise<void>}
+     */
+    _stopStream(jitsiTrack) {
+        if (jitsiTrack) {
+            jitsiTrack.dispose();
```

Comment:
@jallamsetty1 can you confirm this is correct?

Reply:
> @jallamsetty1 can you confirm this is correct?

Working without stopStream()